src/udev/udevadm-trigger.c

```diff
@@ -45,13 +45,13 @@ static int exec_list(sd_device_enumerator *e, const char *action, Set **settle_s
 
                 r = write_string_file(filename, action, WRITE_STRING_FILE_DISABLE_BUFFER);
                 if (r < 0) {
-                        bool ignore = IN_SET(r, -ENOENT, -ENODEV);
+                        bool ignore = IN_SET(r, -ENOENT, -EACCES, -ENODEV);
```

Comment:
hmm, so I think we probably should add comments here, so that this isn't changed again without knowing the full picture.

also I'd prefer if we'd ignore all privilege errors here if we ignore one, i.e. use `ERRNO_IS_PRIVILEGE()`.

Hence:

```
bool ignore = 
        ERRNO_IS_PRIVILEGE(r) || /* Ignore permission problems, possibly enforced by some namespace or MAC-based sandboxing */
        IN_SET(r, 
               -ENOENT, /* ignore devices that have no uevent attribute */
               -ENODEV); /* hmm??? (appears to be work-around for buggy driver?) */
```